test/integration/api/event.spec.js

```diff
@@ -184,9 +184,12 @@ test.serial('check history filters', async (t) => {
     return filteredResults.length === obj.count
   })
 
-  const customRangeValuesCheck = _.curry((prop, obj, rangeValues) => {
+  const customRangeValuesCheck = _.curry((prop, obj, rangeValues, isWithinRange) => {
     let filteredResults = results.filter(r => truncateDate(r.createdDate) === obj.day)
-    filteredResults = filteredResults.filter(r => rangeValues.gte <= r[prop] && r[prop] <= rangeValues.lte)
+    filteredResults = filteredResults.filter(r => isWithinRange(
+      prop === 'createdDate' ? obj.day : r[prop],
```

Comment:
This `day` property` applies to history endpoint, right?

It seems that rounding convention is not specified in API reference draft.

Reply:
Yes indeed. The property `day` is the value of the `groupBy`. I should have added a `groupBy` variable so it would be explicit.

I'll explain the rounding convention in the API reference draft.